app/models/RecurringSchedule.ts

```diff
@@ -2,9 +2,9 @@
  * Data structures used for consistently representing time throughout the app.
  */
 
-import { invert } from "utils/arrays";
 import sortBy from "lodash.sortby";
 import minBy from "lodash.minby";
+import { invert } from "../utils/arrays";
```

Comment:
Jest complained about this import although it appeared to be working in source code.